### ISSUE TITLE
Guestbook: Creation Date Broken

### ISSUE THREAD

**Issue by eaquigley** [IQSS/dataverse#2336]
In the guestbook page for this dataverse: https://dataverse.harvard.edu/dataverse/emiguel the Creation Date for the guestbook is showing as January 1, 1970. That cannot be right.

**Comment by pdurbin** [comment]
@jggautier can you please confirm if this is still a problem or not?

**Comment by jggautier** [comment]
Yes, still seeing January 1, 1970 in that dataverse's guestbook.

![screen shot 2017-06-27 at 11 55 33 am](https://user-images.githubusercontent.com/18374574/27597164-99ff77d0-5b2f-11e7-9284-79788d5b973d.png)

I can't tell if this is a problem in other guestbooks on Harvard Dataverse, but late last year I made a couple of guestbooks on demo dataverse, and the date created is December 15, 2015, which is also wrong. I created another guestbook today, and the date is correct.

![screen shot 2017-06-27 at 12 15 27 pm](https://user-images.githubusercontent.com/18374574/27598073-5a8904a6-5b32-11e7-977f-f4513e202dff.png)

(Possibly unrelated, but I don't understand what it means for a guestbook to be created at another dataverse or the root dataverse.)
